Kconfig

```diff
@@ -0,0 +1,99 @@
+mainmenu "IOT2050 Image Configuration"
+
+config KAS_BUILD_SYSTEM
+	string
+	default "isar"
+
+choice
+	prompt "Image type"
+	default IMAGE_EXAMPLE
+
+config IMAGE_EXAMPLE
+	bool "Example image"
+	help
+	  This is the official example image with several tools as well as
+	  Node-RED preinstalled. It comes without a graphical user interface.
+
+config IMAGE_LXDE
+	bool "LXDE example image"
+	help
+	  Based on the example image, this adds an LXDE-based graphical user
+	  interface.
+
+config IMAGE_BOOT
+	bool "Firmware image"
+	help
+	  Build the firmware image that is responsible for booting the device.
+
+	  WARNING: Do not flash this image onto your device unless you know
+	  that it fits AND you have an external flash programmer at hand that
+	  allows to recover. Otherwise you risk to BRICK THE IOT2050!
+
+endchoice
+
+config KAS_INCLUDE_MAIN
+	string
+	default "kas-iot2050-example.yml" if IMAGE_EXAMPLE
+	default "kas-iot2050-lxde.yml" if IMAGE_LXDE
+	default "kas-iot2050-boot.yml" if IMAGE_BOOT
+
+if IMAGE_EXAMPLE || IMAGE_LXDE
+
+comment "Image features"
+
+config PREEMPT_RT
+	bool "Preempt-RT kernel"
+	help
+	  Build the image with a Preempt-RT kernel, rather than the default
+	  non-preemptive one. This can help achieving better real-time
+	  latencies with the device.
+
+config KAS_INCLUDE_RT
+	string
+	default "kas/opt/preempt-rt.yml"
+	depends on PREEMPT_RT
+
+comment "Build options"
+
+config SDK
+	bool "Build SDK"
+	help
+	  Generate an SDK consisting of a cross-compiler and library headers
+	  needed to build applications for the device.
+
+	  Note: This will not generate the image for the device itself.
+
```

Comment:
Could this be a `Image Type` rather than a `Build opt`?

For example, the below combinations should make no difference on the built output:
- `Example Image` + `Build SDK`
- `Example Image` +  `Preempt-RT` + `Build SDK`
- `LXDE Image` + `Build SDK`
- `LXDE Image` +  `Preempt-RT` + `Build SDK`

If I am not fimilar w/ the code, then I would be confused.

Reply:
Practically, the SDK is the same for example vs. LXDE. Preempt-RT, though, should install a different kernel header package. If not, we had a bug.

Reply:
OK, this means that for the whole picture, we actually should build 3 binaries for SDKs, or at least 2 since LXDE is not in our test plan...

Reply:
...or make the SDK build both kernels and include both kernel header packages. Our current SDK is corresponding to the example image, and that uses the regular kernel.

Reply:
That would be better. 

BTW, could we install two kernels in one example image? Does u-boot support multiple kernel such as grub2?

Reply:
Nope, this is not supported u-boot-script. Would take an interactive boot loader - but that makes no sense for an embedded device.

You can always switch the kernel by building and installing the different package, though.